Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Layout/ButtonLayout.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// Place an object in space relative to another object's scale
+    /// Good for responsive buttons that can stretch and object realign
+    /// </summary>
+    [ExecuteInEditMode]
+    public class ButtonLayout : MonoBehaviour
+    {
+        [Tooltip("Where to set this object's center point in relation to the Anchor's center point")]
+        public Vector3 Alignment;
```

Comment:
nit: if these items have tooltips they should also have summaries containing the tooltip. I woud guess that tooltips won't get picked up by docfx.